docs/migration/migrate-to-10.x.md

```diff
@@ -0,0 +1,76 @@
+| ------------------ | --------------------------------------------------------------------- |
+| Accordion          | [Migrate](../../src/components/accordion/migrate-to-10.x.md)          |
+| Breadcrumb         | [Migrate](../../src/components/breadcrumb/migrate-to-10.x.md)         |
+| Button             | [Migrate](../../src/components/button/migrate-to-10.x.md)             |
+| Carousel           | Removed                                                               |
+| Checkbox           | No breaking changes                                                   |
+| Code Snippet       | [Migrate](../../src/components/code-snippet/migrate-to-10.x.md)       |
+| Combobox           | No breaking changes                                                   |
+| Content Switcher   | [Migrate](../../src/components/content-switcher/migrate-to-10.x.md)   |
+| Copy Button        | [Migrate](../../src/components/copy-button/migrate-to-10.x.md)        |
+| Data Table V2      | TODO                                                                  |
+| Data Table         | Removed                                                               |
+| Date Picker        | TODO                                                                  |
+| Dropdown           | [Migrate](../../src/components/dropdown/migrate-to-10.x.md)           |
+| Fab                | Removed                                                               |
+| File Uploader      | [Migrate](../../src/components/file-uploader/migrate-to-10.x.md)      |
+| Footer             | Removed                                                               |
+| Form               | [Migrate](../../src/components/form/migrate-to-10.x.md)               |
+| Inline Loading     | [Migrate](../../src/components/inline-loading/migrate-to-10.x.md)     |
+| Interior Left Nav  | Removed                                                               |
+| Lightbox           | Removed                                                               |
+| Link               | No breaking changes                                                   |
+| List Box           | No breaking changes                                                   |
+| List               | No breaking changes                                                   |
+| Loading            | [Migrate](../../src/components/loading/migrate-to-10.x.md)            |
+| Modal              | [Migrate](../../src/components/modal/migrate-to-10.x.md)              |
+| MultiSelect        | No breaking changes                                                   |
+| Notification       | [Migrate](../../src/components/notification/migrate-to-10.x.md)       |
+| Number Input       | [Migrate](../../src/components/number-input/migrate-to-10.x.md)       |
+| Overflow Menu      | [Migrate](../../src/components/overflow-menu/migrate-to-10.x.md)      |
+| Pagination Nav     | Removed                                                               |
```

Comment:
Pagination nav is a new component that is introduced very recently, instead of deprecated one.

```suggestion
```